packages/rs-drive-abci/src/platform_types/platform_state/v0/mod.rs

```diff
@@ -784,4 +805,104 @@ impl PlatformStateV0Methods for PlatformStateV0 {
     )> {
         self.previous_height_chain_lock_validating_quorums.as_ref()
     }
+
+    fn full_masternode_list_changes(&self, previous: &PlatformStateV0) -> MasternodeListChanges {
```

Comment:
Not sure it it matters, but this logic ignores any changes that happened between two states. Maybe we should check if previous is really previous (h-1) and not very old one?

Reply:
No, it does, what it's supposed to do, this can be used for distant states if we want. Though in practice that won't happen yet.